webapp/sectionSpec/rowSpec.ts

```diff
@@ -15,9 +15,9 @@ export interface RowSpec {
   calculateFn?: CalculateValue
   chartProps?: RowChartSpec
   idx?: string | number
-  // labelKey?: string
-  // labelPrefixKey?: string
-  // labelParams?: Record<string, string>
+  labelKey?: string
```

Comment:
Let's add now an object label with properties key, params, props
Btw where is it needed ?